### ISSUE TITLE
Media Gallery Click event not open dialog box to add alt text and other inputs

### ISSUE THREAD

**Issue by vikaskalal** [magefan/module-blog#580]
Hello Team,

it's regarding MageFan Blog module we are using your free megafan blog module in our website and i have encountered one issue when we are creating a new post there is one media gallery section in which the click event is not working. This means after uploading the image when I click on the image there is nothing opened and it does not give the option to add alt text.

**Comment by magefan** [comment]
@vikaskalal , yes, unfortunatly there is no such option to add alt to the blog post gellery image.
